tests/std/tests/usual_matrix.lst

```diff
@@ -9,12 +9,14 @@ RUNALL_INCLUDE .\prefix.lst
 RUNALL_CROSSLIST

```

Comment:
> I haven't kept track of what's happened in prod/fe internally - if /std:c++20 was never removed there, then we're OK for internal mirroring.

I've already mirrored this and successfully built and ran `contest stl`, so it appears that `/std:c++20` is going to be removed only from `rel/16.10`.

> Idea: Could we use the Python harness's ability to mark "features" as supported, so that any configurations with /std:c++20 can be centrally disabled with a single line, allowing contributors to update to 16.10 GA? Then the first toolset update supporting /std:c++20 can change that single line.

Yes. I can hack our custom test format to treat `/std:c++20` as a requirement for an unsupported feature. The tests will be skipped in GitHub CI but continue to run internally. (This is an order of magnitude improvement vs. the alternative of listing all such configurations as `SKIPPED` in `expected_results.txt`.) I'll do this.

Reply:
I manually verified the `test.py` change with both a `std` and a `tr1` test by checking that the number of `UNSUPPORTED` configurations is lower with the change commented out than with it active.